TaskClient.cs

```diff
@@ -80,6 +82,7 @@ public CancellationTokenSource Start()
                     }
                     
                     await ExecuteQueuedTask();
+                    Thread.Sleep(_pollDelay);
```

Comment:
Use Task.Delay for more efficient use of the thread pool.

Reply:
Thanks for the suggestion, I didn't know about Task.Delay.